cli/script/command-executor.ts

```diff
@@ -1020,6 +1056,49 @@ function getPackageMetricsString(packageObject: PackageWithMetrics): string {
     return returnString;
 }
 
+function getReactNativeProjectAppVersion(platform: string, projectName: string): Promise<string> {
+    if (platform === "ios") {
+        try {
+            var infoPlistContents: string = fs.readFileSync(path.join("iOS", projectName, "Info.plist")).toString();
+        } catch (err) {
+            try {
+                infoPlistContents = fs.readFileSync(path.join("iOS", "Info.plist")).toString();
+            } catch (err) {
+                throw new Error(`Unable to find or read "Info.plist" in the "iOS/${projectName}" or "iOS" folders.`);
+            }
+        }
+        
+        var parsedInfoPlist: any = plist.parse(infoPlistContents);
+        if (parsedInfoPlist && parsedInfoPlist.CFBundleShortVersionString) {
+             if (semver.valid(parsedInfoPlist.CFBundleShortVersionString) === null) {
+                throw new Error("Please update \"Info.plist\" to use a semver-compliant \"CFBundleShortVersionString\", for example \"1.0.3\".");
```

Comment:
@lostintangent - should we just keep 1.0.0 as the default value in example/error/help messages?